msi/release.sh

```diff
@@ -152,8 +152,10 @@ die "Could not generate file list"
 # Write the GitComponents.wxs file containing the file list
 SCRIPT_WINPATH="$(cd "$SCRIPT_PATH" && pwd -W | tr / \\\\)"
 BUILD_EXTRA_WINPATH="$(cd "$SCRIPT_PATH"/.. && pwd -W | tr / \\\\)"
+BIND_IMAGE_PATHS='[INSTALLFOLDER]usr\\bin;[MingwFolder]bin;[System$(var.SixtyFourBit)Folder]'
```

Comment:
Hmm, yes. We could define a `<Property>` and then reference just the property ID in the `@BindPath` attribute.